src/qt/uicomponents/TrackType.h

```diff
@@ -0,0 +1,24 @@
+#ifndef TRACK_TYPE_H
+#define TRACK_TYPE_H
+
+#include <QtCore/QtCore>
+#include <QtQml/qqmlregistration.h>
+
+class TrackType
```

Comment:
We will need to figure out how to make it open ended. No assumption should be made about the available track types...